bin/strings.swift

```diff
@@ -121,7 +121,7 @@ let stringsByLocale = stringsByLocale1
 }
 
 stringsByLocale?.forEach { locale, strings in
-  guard locale != "it" else { return }
```

Comment:
Should we instead do a check that `locale` is contained within a list of our supported locales? That way if kickstarter adds another locale that we don't have a folder for, we don't have to update this again?

Something like:
```
let supportedLocales = ["en", "es", ...]
stringsByLocale?.forEach { locale, strings in
    guard supportedLocales.contains(locale) else { return }
    ...
}
```